client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -67,6 +67,7 @@ export interface SiteNode {
 
 export interface StatsNode {
 	type: AllowedTypes;
+	status: AllowedStatusTypes | string;
```

Comment:
I think we can keep this as `status: AllowedStatusTypes` since we have added 'active` as an allowed type.